uint32 select # select obj at this index
Target[] objects
---
bool success
string info
